convert2rhel/utils.py

```diff
@@ -533,12 +533,14 @@ def backup(self):
 
 
 def require_single_user_mode():
-    """
-    Make sure that the system is running in single-user mode and exit if it is not.
+    """Make sure that the system is running in single-user mode and exit if it is not.
+
+    If booted into multi-user mode, a different logged in user than the one executing the conversion may be interfering
+    with the system during the conversion, potentially leaving the system in an undefined state.
     """
     loggerinst = logging.getLogger(__name__)
 
-    output, ret_code = run_cmd_in_pty("runlevel", print_cmd=False)
+    output, ret_code = run_cmd_in_pty("runlevel", print_cmd=True, print_output=False)
```

Comment:
This will not be necessary after https://github.com/oamg/convert2rhel/pull/149  merged.